oat-sa/extension-lti-outcomeui
# Feature/MS-1185/remove all highlights button

**Related to task:** https://oat-sa.atlassian.net/browse/MS-1185
**Description:** Add button to remove all highlights at once. 
Align highlights control panel [with the latest design](https://projects.invisionapp.com/share/JGULEID89F2#/screens/433499500_component_-_Highlighter_Tool_-_UI)

Before | After
-------|-----------
<img width="330" alt="Screenshot 2021-06-16 at 10 34 20" src="https://user-images.githubusercontent.com/6339002/122188448-3ed9b680-ce90-11eb-9b76-da8bcbb65c6c.png"> | <img width="509" alt="Screenshot 2021-06-16 at 10 40 46" src="https://user-images.githubusercontent.com/6339002/122188315-210c5180-ce90-11eb-935a-7382d39a2283.png">

**Depends on:** 
- [ ] https://github.com/oat-sa/tao-core-ui-fe/pull/311

**How to test:** 
- Login to MS as scorer
- Use highlighter feature
- Click on "Clear all"  button should delete all highlights and reset the colors counters
